src/utils/noop.ts

```diff
@@ -0,0 +1 @@
+export default function noop() {}
```

Comment:
helpful when we wanna default a prop to a noop function, without creating a new function instance within the component